src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,262 @@
+package duke;
+
+import duke.command.Deadline;
+import duke.command.Event;
+import duke.command.Todo;
+import duke.task.Task;
+
+import java.nio.file.Path;
+import java.util.Objects;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.nio.file.Paths;
+
+public class Duke {
+
+    public static int taskNum = 0;
+    public static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static final String root = System.getProperty("user.dir");
+    public static final Path dataPath = Paths.get(root, "data", "duke.txt");
+    public static final Path dirPath = Paths.get(root, "data");
+
+    public static void main(String[] args) {
+        String userCommand;
```

Comment:
Good job with applying the [SLAP principle](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) here!